descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "orders",
-      "version": "12.0",
+      "version": "12.1",
```

Comment:
As I remember if we increase from 12.0 to 12.1 - Okapi will not throw an error, even if in mod-gobi required version is 12.0 https://github.com/folio-org/mod-gobi/blob/master/descriptors/ModuleDescriptor-template.json#L157 
If we for example we increased from 12.0 to 13.0 and in mod-gobi remain 12.0 - Okapi will throw an error during enabling the module. 
But you can check my words using karate tests(acquisition folder in folio-integration-test) or just locally.

Basically we not always follow the rule to update minor version when adding new field, but in case when you increased it also would be a good practice to increase in these modules as well:
https://github.com/folio-org/mod-invoice/blob/master/descriptors/ModuleDescriptor-template.json#L664
https://github.com/folio-org/mod-gobi/blob/master/descriptors/ModuleDescriptor-template.json#L157
https://github.com/folio-org/mod-ebsconet/blob/master/descriptors/ModuleDescriptor-template.json#L6

Reply:
Basically I am ok to keep major versions for example in mod-gobi 12.0 and in orders increase to 12.1. It should work